Simplified/StringUtilities.m

```diff
@@ -0,0 +1,26 @@
+//
+//  StringUtilities.m
+//  Simplified
+//
+//  Created by Aferdita Muriqi on 6/20/17.
+//  Copyright © 2017 NYPL Labs. All rights reserved.
+//
+
+#import "StringUtilities.h"
+
+@implementation StringUtils
```

Comment:
Since this is an Objective-C class that's part of the app itself, we should be sticking with the established `NYPL___` naming convention, e.g. `NYPLStringUtilities`.